tests_config/run_spec.js

```diff
@@ -9,6 +9,7 @@ const normalizeOptions = require("prettier/src/main/options").normalize;
 const AST_COMPARE = process.env["AST_COMPARE"];
 
 function run_spec(dirname, parsers, options) {
+  global.isInTest = true;
```

Comment:
can you remove this change from this PR and made another one just with it?

There rest looks fantastic!

Reply:
Sure thing!